ros_buildfarm/templates/snippet/builder_system-groovy_check-for-failing-jobs.xml.em

```diff
@@ -0,0 +1,82 @@
+@(SNIPPET(
+    'builder_system-groovy',
+    command=
+"""// CHECK FOR FAILING JOBS
+import hudson.model.Job
+
+// Control the number of failures in a row to check
+min_number_of_failures = 5
+// Control the minimun amount of time failing
+min_days_failing = 7
+// rosdistro to search
+rosdistro = @ros_distribution
```

Comment:
You called the variable `ros_distro` (https://github.com/ros-infrastructure/ros_buildfarm/pull/454/files#diff-f331a1152ef526adbb9d081d5e96a1b8R52) not `ros_distribution`. So you have to use the same name here. And since it is a string you will need quotes around it.